api/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use LAMP image
+FROM mattrayner/lamp:latest-1804
```

Comment:
Should probably `latest` instead of `latest-1804`, unless you have a particular need for that version.

Reply:
done